### PR TITLE
Add doc for status api

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
First time, so please be gentle ;-)

### CODE REVIEW

File: developer_manual/client_apis/OCS/ocs-status-api.rst [hunk 1]
```diff
@@ -0,0 +1,152 @@
+===========
+OCS Status API
+===========
```
Comment (ChristophWurst) [inline developer_manual/client_apis/OCS/ocs-status-api.rst:3]: make the `=` as long as the heading :)

File: developer_manual/client_apis/OCS/ocs-status-api.rst [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+The OCS Status API allows you to access and modify status API from outside over pre-defined OCS calls.
+
+The base URL for all calls to the share API is: *<nextcloud_base_url>/ocs/v2.php/apps/user_status/api/v1/user_status*
+
+All calls to OCS endpoints require the ``OCS-APIRequest`` header to be set to ``true``.
+
+
+User Status - Status Manipulation
+------------
```
Comment (ChristophWurst) [inline developer_manual/client_apis/OCS/ocs-status-api.rst:13]: same

File: developer_manual/client_apis/OCS/ocs-status-api.rst [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+Fetch your own status
+^^^^^^^^^^^^^^
```
Comment (ChristophWurst) [inline developer_manual/client_apis/OCS/ocs-status-api.rst:16]: same

File: developer_manual/client_apis/OCS/ocs-status-api.rst [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+* Required capability: `user_status`
+* Method: `GET`
```
Comment (ChristophWurst) [inline developer_manual/client_apis/OCS/ocs-status-api.rst:19]:
```suggestion
* Method: ``GET``
```

for monospace in sphinx. also fix lines above and below

**Review by ChristophWurst — COMMENTED** [review]
* Heading symbol lines are always as long as the text
* Use double back ticks for monospace in shinx

**Review by ChristophWurst — APPROVED** [review]
Looks good :)

**Comment by ChristophWurst** [comment]
backport to 20?

**Comment by tobiasKaminsky** [comment]
> backport to 20?

I would say yes, as it was introduced in NC20.

**Comment by ChristophWurst** [comment]
then sure :)

File: developer_manual/client_apis/OCS/ocs-status-api.rst [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+^^^^^^^^^^^^^
+
+The user-status is also exposed via the following Files Sharing APIs:
+ * ``GET /ocs/v2.php/apps/files_sharing/api/v1/sharees``
+ * ``GET /ocs/v2.php/apps/files_sharing/api/v1/sharees_recommended``
+ * ``GET /ocs/v2.php/apps/files_sharing/api/v1/shares``
+ * ``GET /ocs/v2.php/apps/files_sharing/api/v1/shares/inherited``
+ * ``GET /ocs/v2.php/apps/files_sharing/api/v1/shares/pending``
+ * ``GET /ocs/v2.php/apps/files_sharing/api/v1/shares/{id}``
+ * ``POST /ocs/v2.php/apps/files_sharing//api/v1/shares``
```
Comment (nickvergessen) [inline developer_manual/client_apis/OCS/ocs-status-api.rst:151]:
```
 * ``POST /ocs/v2.php/apps/files_sharing/api/v1/shares``
```

**Pull request merged**